src/App.js

```diff
@@ -0,0 +1,25 @@
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
```

Comment:
You can remove most of the code inside this component and only leave a placeholder or nothing. It's not worth checking this stuff in if its eventually going to be deleted.